utils/workflow.go

```diff
@@ -76,3 +78,19 @@ func GetConfigMap(client client.Client) (corev1.ConfigMap, error) {
 		return existingConfigMap, nil
 	}
 }
+
```

Comment:
yes, I think that it is a good idea renaming it to something like GetCommonConfigForBuilder because this method is the one we use to retrieve the config map holding the common configuration for the builder.

Since it is also a method used by the Build controller I would lie to move in a specific file. What do you think?

Reply:
sounds good :)